gradle/libs.versions.toml

```diff
@@ -51,8 +51,8 @@ gradlePlugin-gradlePluginPublish = "0.20.0"
 gradlePlugin-gradleNode = "3.5.1"
```

Comment:
JFYI I also didn't touch anything related to publishing to avoid breaking something. These versions and dependencies should be revisited as part of #3132